### PR TITLE
Removed apache license from doc files and added 3.1.0 release changes

### PR DESCRIPTION
**Pull request by meenakshidembi691**
# Description
Removed apache license from doc files

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [ ] I have performed Ansible Sanity test using --docker default
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Backward compatibility is not broken

### CODE REVIEW

File: roles/capacity_role/README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Example Playbook
 License
 -------
 
-Apache 2
+GPL
```
Comment (sachin-apa) [inline roles/capacity_role/README.md:46]:
```suggestion
GPL-3.0 license
```

**Review by sachin-apa — APPROVED** [review]

**Review by sachin-apa — APPROVED** [review]

**Review by Bhavneet-Sharma — APPROVED** [review]

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,13 +1,3 @@
...
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
--->
-
 # How to contribute
 
 Become one of the contributors to this project! We thrive to build a welcoming and open community for anyone who wants to use the project or contribute to it. There are just a few small guidelines you need to follow. To help us create a safe and positive community experience for all, we require all participants to adhere to the [Code of Conduct](https://github.com/dell/ansible-powermax/blob/3.0.0/docs/CODE_OF_CONDUCT.md).
```
Comment (trisha-dell) [inline docs/CONTRIBUTING.md:3]: Update the version in link.

**Review by trisha-dell — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -33,35 +33,35 @@ This table provides information about the software prerequisites for the Ansible
 
 | **Ansible modules** | **Unisphere version** | **PowerMaxOS** | **Python version**            | **Python library version** | **Ansible**              |
 |---------------------|-----------------------|----------------|-------------------------------|----------------------------|--------------------------|
-| v3.0.0 | 9.2 <br> 10.0 <br> 10.1 | 5978.444.444 <br> 5978.669.669 <br> 5978.711.711 <br> 6079.xxx.xxx | 3.9.x <br> 3.10.x <br> 3.11.x | 9.2.x.x <br> 10.0.x.x <br> 10.1.x.x | 2.14 <br> 2.15<br> 2.16 |
+| v3.1.0 | 9.2 <br> 10.0 <br> 10.1 | 5978.444.444 <br> 5978.669.669 <br> 5978.711.711 <br> 6079.xxx.xxx | 3.9.x <br> 3.10.x <br> 3.11.x | 9.2.x.x <br> 10.0.x.x <br> 10.1.x.x | 2.14 <br> 2.15<br> 2.16 |
```
Comment (trisha-dell) [inline README.md:36]: update Ansible version to 2.15, 2.16 and 2.17

File: docs/ISSUE_TRIAGE.md [hunk 4]
```diff
@@ -43,7 +33,7 @@ Bug reports should explain what happened, what was expected and how to reproduce
 
  - Ansible Version: [e.g. 2.14]
```
Comment (Bhavneet-Sharma) [inline docs/ISSUE_TRIAGE.md:34]:
```suggestion
 - Ansible Version: [e.g. 2.15]
```

File: README.md
Comment (sachin-apa) [inline README.md:1]: Readme format needs to be updated as per the latest ansible latest guidline, please refer powerstore repo.

**Review by sachin-apa — APPROVED** [review]

**Review by Bhavneet-Sharma — APPROVED** [review]

**Pull request merged**